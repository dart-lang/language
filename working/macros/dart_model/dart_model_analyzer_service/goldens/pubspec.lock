# Generated by pub
# See https://dart.dev/tools/pub/glossary#lockfile
packages: {}
sdks:
  dart: ">=3.4.0 <4.0.0"
