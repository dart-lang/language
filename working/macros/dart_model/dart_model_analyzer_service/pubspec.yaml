name: dart_model_analyzer_service
publish-to: none

environment:
  sdk: ^3.4.0

dependencies:
  analyzer: '>=5.2.0 <7.0.0'
  dart_model: any
  pool: ^1.5.1
  stream_transform: any

dev_dependencies:
  json_schema: ^5.1.7
  test: ^1.25.7

dependency_overrides:
  dart_model:
    path:
      ../dart_model
