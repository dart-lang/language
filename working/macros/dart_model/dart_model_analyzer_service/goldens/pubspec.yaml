name: goldens
publish-to: none

environment:
  sdk: ^3.4.0
